extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/CriterionDto.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (c) 2022 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.ext.wrapper.api.ui.model;
+
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class CriterionDto {
+    private String operandLeft;
+    private OperatorDto operator;
+    private CriterionLiteralDto operandRight;
+
+    public static CriterionDto forString(String operandLeft, OperatorDto operator, CriterionLiteralDto criterionLiteral) {
+        return new CriterionDto(operandLeft, operator, criterionLiteral);
+    }
+
+    public static CriterionDto forJson(String operandLeft, OperatorDto operator, CriterionLiteralDto operatorRight) {
+        return new CriterionDto(operandLeft, operator, operatorRight);
+    }
```

Comment:
these functions should be removed from here.

These were originally utility methods for a sum type. This class is not a sum type anymore.

The CriterionLiteralDto is a sum type. 

The utility methods are for instantiating the sum type easier

```java
@AllArgsConstructor
class MySumType {
  MySumTypeType type;
  A a;
  B b;
  C c;

  // the following methods are purely for convenience

  public static ofA(A a) {
    return MySumType(MySumType.OPTION_A, a, null, null);
  }

  public static ofB(B b) {
    return MySumType(MySumType.OPTION_B. null, b, null);
  }

  public static ofC(C c) {
    return MySumType(MySumType.OPTION_C. null, null, c);
  }
}
```